### ISSUE TITLE
v2-dev: implement `resetOthers` similar to .reset/.clear methods

### ISSUE THREAD

**Issue by KES777** [brainfoolong/form-data-json#14]
**Is your feature request related to a problem? Please describe.**
We have `clear`, `reset` methods:
```
FormDataJson.reset(document.querySelector("form"))
FormDataJson.clear(document.querySelector("form"))
```

But only one option: `clearOthers`

**Describe the solution you'd like**
Impelement `resetOthers` option for usage with `FormDataJson.fromJson( ... )`

**Comment by brainfoolong** [comment]
Yep, didn't added it because i didn't saw a real world usage for this. But yes, it's just a few lines to add that feature.

**Comment by brainfoolong** [comment]
Thx for request. Now implemented in 2.0.3beta

**Comment by KES777** [comment]
Did not check functionality, yet, but I see: option is implemented. Thank you
